src/Commands/App/Builder.php

```diff
@@ -100,8 +100,16 @@ protected function compile(string $name): Builder
             ->getCompiler($name);
 
         $structure = config('app.structure') ?: $this->structure;
+        
+        // See - https://github.com/laravel-zero/framework/pull/102
+        $regex = '#'.implode('|', $structure).'#';
 
-        $compiler->buildFromDirectory(BASE_PATH, '#'.implode('|', $structure).'#');
+        if (stristr(PHP_OS, "WINNT") != FALSE) {
```

Comment:
Should use strict comparison here: `!==`. Otherwise it has the possibility of false positives in some edgecases.

Reply:
good point! thanks.

Reply:
@lesstif Can you fix it? I can do it for you if you want 🙂

Reply:
@nunomaduro i fixed it. thanks.